<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApsCriptografia</class>
 <widget class="QWidget" name="ApsCriptografia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="Entrada_Texto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Crip">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>CRIPTOGRAFAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Des">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>DESCRIPTOGRAFAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="fundo">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>1080</width>
     <height>675</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../APS Cripto/criptografia-1080x675.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../20x20.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Trabalho APS Unip Anchieta Prof: Marcos .Jr Desenvolvido por Bruno kkkk, Cristian Rodriguez Passos e Gustavo assisss</string>
   </property>
  </widget>
  <zorder>fundo</zorder>
  <zorder>Entrada_Texto</zorder>
  <zorder>label</zorder>
  <zorder>Btn_Crip</zorder>
  <zorder>Btn_Des</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
